demoproj/settings.py

```diff
@@ -109,18 +103,11 @@
         },
     },
     'loggers': {
-        'django': {
-            'handlers': ['console'],  # <-- Specify handlers
-            'level': 'INFO'
-        },
-        'demoproj': {
-            'handlers': ['console'],  # <-- Specify handlers
-            'level': 'DEBUG'
+        'django': {'handlers': ['console'], 'level': 'INFO'},  # <-- Specify handlers
```

Comment:
Hmmm. Can we allow multiline JSON? (Maybe only in this file) `DJANGO_GUID` above also feels less readable with this style.

Reply:
I agree configurations like these are more readable when expanded. Unfortunately, I don't know that it is possible to configure black to this extent.

It is possible to exclude files by adding them to `exclude` in .black

Reply:
Excluded the settings in 2f347fb5546690b3ce78230d687fa330456c7bc9